packages/react/alert-dialog/src/AlertDialog.tsx

```diff
@@ -47,112 +34,44 @@ const [
   useAlertDialogContentContext,
 ] = createContext<AlertDialogContentContextValue>(CONTENT_NAME);
 
-type AlertDialogContentOwnProps = Omit<
-  Polymorphic.OwnProps<typeof DialogPrimitive.Content>,
-  'refToFocusOnOpen' | 'id'
->;
+type AlertDialogContentOwnProps = Omit<Polymorphic.OwnProps<typeof DialogPrimitive.Content>, 'id'>;
```

Comment:
I know you haven't changed this but curious if you know why we remove the `id` prop here? I don't think we do this anywhere else, we sorta just allow people to break things.

Reply:
Yeah true, I didn't give this one much thought, I'll check it and compare with others.

Reply:
Yeah totally right, we usually allow all overriding, plus there was no reason to remove it here ever cause we were not doing anything different with `id` in `AlertDialogContent` specifically 🤷‍♂️